src/main/java/org/fundacionjala/virtualassistant/models/RequestEntity.java

```diff
@@ -0,0 +1,57 @@
+package org.fundacionjala.virtualassistant.models;
+
+import javax.persistence.*;
```

Comment:
again please use specfic class names not wildcards 
<img width="425" alt="Screenshot 2023-08-04 at 18 24 49" src="https://github.com/USB-VirtualAssistant/virtualAssistant/assets/6982690/482d40fe-2847-4936-a766-b93093da3d46">